installer_show_maps_data/prep_data_for_maps.py

```diff
@@ -66,7 +66,56 @@
 
 
 # %% [markdown]
-# ## Filtering data
+# ## 3. Loading household data per local authority
+#
+# Household info for Scotland here:
+#
+# Household info for England and Wales here:
+# https://www.ons.gov.uk/datasets/TS041/editions/2021/versions/3
+#
+#
+# Data was downloaded from the websites and then uploaded to S3. Before uploading to S3 a few LA names were updated to match the names in the MCS data.
+
+# %%
+scotland_la_households = pd.read_excel(
+    "s3://asf-exploration/installer_show_maps_data/inputs/scotland_households.xls"
+)
+england_wales_la_households = pd.read_csv(
+    "s3://asf-exploration/installer_show_maps_data/inputs/england_wales_households.csv"
+)
+
+
+# %%
+scotland_la_households.head()
+
+# %%
+england_wales_la_households.head()
+
+# %%
+scotland_la_households.columns
+
+# %%
+scotland_la_households = scotland_la_households[
+    ["Area Name", "Area code", "2022"]
+].rename(
+    columns={"Area Name": "LA", "Area code": "LA Code", "2022": "Number of Households"}
+)
+england_wales_la_households = england_wales_la_households.rename(
+    columns={
+        "Lower Tier Local Authorities": "LA",
```

Comment:
I found this online: _"In some areas of England, local government is divided between a county council (upper-tier) and a district council (lower-tier), which are responsible for different services. In other areas, there is a single unitary authority instead. In Scotland, Wales and Northern Ireland there are only unitary, single-tier councils."_

Definitely not an expert in UK geographies, but yeah... lower tier seem to be the correct ones, but i call them just LAs for simplicity :)

Reply:
Thanks for looking that up :) I think it's fine because it matches in flourish like we said and it's just a preference i guess of which level we want to show it to. Lower tier gives us more granularity